rollingpin/elasticsearch.py

```diff
@@ -0,0 +1,48 @@
+import time
+import json
+
+from twisted.internet import reactor
+from twisted.web.client import Agent
+from twisted.web.http_headers import Headers
+from twisted.web.test.test_newclient import StringProducer
+
+
+class ElasticSearchNotifier(object):
+    def __init__(self, config, components):
+        base_url = config["elasticsearch"]["endpoint"]
+        index, type = config["elasticsearch"]["index"], config["elasticsearch"]["type"]
```

Comment:
it seems unnecessary to do this tuple packing/unpacking here rather than just having three lines for base_url, index, and type respectively. (also, `type` is shadowing a built-in, maybe `event_type` or something?)